packages/deploy-utils/src/setup-credentials/script.ts

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S vite-node --script
+
+import { setupCredentials } from ".";
+import { isTarget } from "../utils/types";
+
+const target = process.argv[2];
+if (!isTarget(target)) {
```

Comment:
maybe allow the `isTarget` function parameter to be `string | undefined` in the function definition. I see a few places where we apply `?? ""` to the parameter